automated/health-checks/tests/health-check-002.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+TEST_CASE_NAME=$(basename "$0" .sh)
+
+# Check for errors and warnings in dmesg
+if dmesg | grep -E 'error|fail|warning' >/dev/null; then
+    lava-test-case "$TEST_CASE_NAME-Errors or Warnings in dmesg: Found" --result fail
+else
+    lava-test-case "$TEST_CASE_NAME-Errors or Warnings in dmesg: Not Found" --result pass
+fi
+
+# Check CPU load
+cpu_load=$(uptime | awk -F 'load average:' '{print $2}' | awk -F, '{print $1}' | awk '{$1=$1};1')
+echo "CPU Load: $cpu_load"
+
+# Check available memory
+free_mem=$(free -m | awk 'NR==2{printf "%.2f%%", $3*100/$2}')
+echo "Available Memory: $free_mem"
+
+# Check disk space
+disk_space=$(df -h / | awk 'NR==2{printf "%s", $5}')
+echo "Disk Space: $disk_space"
```

Comment:
Is this just general information we'd like to have in the test? If so, is it needed/nice to have, or is it contributing to the LAVA noise?

Reply:
What is nice to have in a health check?
`
A test job for one specific device type which is automatically run at regular intervals to ensure that the physical device is capable of performing the minimum range of tasks. If the health check fails on a particular device, LAVA will automatically put that device offline. Health checks have higher priority than any other jobs.
`
[health check](https://lava.revpi.kunbus.com/static/docs/v2/glossary.html#term-health-check)

Reply:
That was my question to you :p

Is this information useful for the health check or can we remove this? Personally, I'm not sure what this information can be useful for, but you might disagree on that so I'm asking.